libraries/botbuilder-ai-orchestrator/src/orchestratorAdaptiveRecognizer.ts

```diff
@@ -51,7 +52,9 @@ type Orchestrator = {
 /**
  * Class that represents an adaptive Orchestrator recognizer.
  */
-export class OrchestratorAdaptiveRecognizer extends Recognizer implements OrchestratorAdaptiveRecognizerConfiguration {
+export class OrchestratorAdaptiveRecognizer
```

Comment:
See class diagram in https://github.com/microsoft/botbuilder-js/issues/3295#issue-805199506

Decided to keep things in line with JS and C#, by making `OrchestratorAdaptiveRecognizer` derive from `AdaptiveRecognizer`, which is what we want to have all adaptive recognizers do in the first place. This means that we needed to:
* Make botbuilder-ai-orchestrator take dependency on botbuilder-dialogs-adaptive
* This is how it is in C# already anyways
* Additionally Orchestrator is in preview, so if we want to make changes to align the languages, best to do them now

___

Alternatively, if we did not want to change the dependency of Orchestrator, then, we could make it follow the path that requires more duplication in the SDK, but at least no dependency change
* OrchestratorAdaptiveRecognizer would house its own logPersonalInformation property + have its own override of fillRecognizerResultTelemetryProperties